components/ui/Window.js

```diff
@@ -42,19 +43,28 @@ const Title = styled.div`
   font-weight: normal;
   font-size: 32px;
   line-height: 41px;
-  padding: 16px;
+  padding: 13px 20px;
```

Comment:
Hmmm... rather than assigning padding here, I don’t think we’d ever want this to wrap if the window thin and title long. It’s probably better to set a fixed height (or inherit) the title bar height of 6.6rem and then overflow: hidden with a text-ellipsis, since that would be more akin to a OS titlebar. And then you could just let the flex handle the centering.

Reply:
removed top/bottom padding, also added ellipsis for the text